### PR TITLE
Fixed flow errors on a few 'view.jsx' files and 'form-field.jsx'

### PR DESCRIPTION
**Pull request by dannycalleri**
Fixed a few flow errors, connected to this issue:
https://github.com/lbryio/lbry-desktop/issues/2024 .
Let me know what you think 🤘

### CODE REVIEW

File: src/renderer/component/channelTile/view.jsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ type Props = {
   claim: ?{
     claim_id: string,
     name: string,
+    permanent_url: string,
```
Comment (seanyesmunt) [inline src/renderer/component/channelTile/view.jsx]: This should use the `Claim` type in `types/claim`

Comment (dannycalleri) [reply src/renderer/component/channelTile/view.jsx]: Sorry, didn't notice that @seanyesmunt  :)

Comment (dannycalleri) [reply src/renderer/component/channelTile/view.jsx]: Changed 🤘

**Comment by seanyesmunt** [comment]
Thanks for the PR! Can we [send you some LBC as appreciation](https://lbry.io/faq/appreciation)?

**Comment by tzarebczan** [comment]
@dannycalleri If you'd like a Hacktoberfest t-shirt from LBRY, please reach out to us at the same address linked above!

**Pull request merged**